packages/quill-cdn/assets/documents/quill_sitemap.xml

```diff
@@ -0,0 +1,3508 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Was this sitemap.xml generated manually or was some tool used?  

It might be worth adding to the How Section for future updates.

Reply:
added!